instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/middleware/otel_middleware.py

```diff
@@ -208,7 +223,18 @@ def process_request(self, request):
         )
 
         attributes = collect_request_attributes(carrier)
-
+        active_requests_count_attrs = _parse_active_request_count_attrs(
+            attributes
+        )
+        duration_attrs = _parse_duration_attrs(attributes)
+        request_start_time = default_timer()
```

Comment:
Should this timer start after the tracing logic? Wouldn't it be inaccurate to include the time spent on span recording  in the request duration? I haven't paid attention to this detail for prev instrumentations. I think we should check them as well in a separate issue.

Reply:
ok got it. I will start the timer after the span is recorded